images/macos/provision/configuration/configure-machine.sh

```diff
@@ -17,7 +17,9 @@ if [ -d "/Library/Application Support/VMware Tools" ]; then
     sudo "/Library/Application Support/VMware Tools/vmware-resolutionSet" 1176 885
 fi
 
-# Update VoiceOver Utility to allow VoiceOver to be controlled with AppleScript by creating a special file (SIP must be disabled)
+# Update VoiceOver Utility to allow VoiceOver to be controlled with AppleScript by updating defaults and creating a special file (SIP must be disabled)
+defaults write com.apple.VoiceOver4/default SCREnableAppleScript -bool YES
```

Comment:
Happy to, but may I ask why? As far as I’m aware this command updating defaults doesn’t require SIP to be disabled, so guarding it behind the `csrutil status` check could be misleading. Is it simply to keep the two VO commands in lockstep? Either you get both or neither?

Reply:
@cmorten , Oh, I see. In that case, could you add a comment to describe why we need this command? Because currently the comment above contains `(SIP must be disabled)`  description.

Reply:
Ah yes, that description is now somewhat misleading, I’ll have a think and push something up!